### ISSUE TITLE
[BUG] Can't give pills to buckled/surgically open patients.

### ISSUE THREAD

**Issue by AGasCanister** [DS-13-Dev-Team/DS13#416]
#### Description of issue

When a person is buckled to something or surgically open you won't be able to interact with him, meaning that you won't be able to give him pills. (This also happened with defibs)

#### Difference between expected and actual behavior

The idea is that you should be able to give him pills either if he's buckled or surgically open

#### Steps to reproduce 
1. Be a human and get another human.
2. Buckle the human
3. get a pill or surgically open him
4. you won't be able to give him the pill.

#### Specific information for locating

Human, pill, anything that you can be buckled into, scalpel.

#### Length of time in which bug has been known to occur

Basically the last rounds of 7th of June, 2020

#### Client version, Server revision & Game ID

Can't get it because can't get into server, server revision and game ID of 7th of June, 2020

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/DS-13-Dev-Team/DS13/issues)
